### DISCUSSION TITLE
change the order of icons in the component menu

### DISCUSSION THREAD

**Discussion by guibeutler** [ant-design/ant-design#43632] [General]
hello everyone, I need to change the default menu, which is presented as follows:

![image](https://github.com/ant-design/ant-design/assets/101596777/884e68d2-e432-41cd-a927-8da6238ee64a)

I need it to be in the following order:

![image](https://github.com/ant-design/ant-design/assets/101596777/93078d57-736a-4e72-a973-6b9ba039b20b)

I'm using the component as follows:

![image](https://github.com/ant-design/ant-design/assets/101596777/33de2bfb-5c25-467c-834d-1fe006905984)

Is it possible to change directly in the component or with css? can you help?

**Comment by wving5** [comment]
Oops, seems already resolved
~~given it's a flex layout, CSS `order` property can be utilized, like such workaround~~
```
  .ant-menu-submenu-title {
    span:nth-last-child(1) {
      order: -1;
    }
  }
```
